src/api.v2/helpers/pipeline/pipelineConstruct/index.js

```diff
@@ -176,6 +205,8 @@ const createQCPipeline = async (experimentId, processingConfigUpdates, authJWT,
     {
       state_machine_arn: stateMachineArn,
       execution_arn: executionArn,
+      last_pipeline_params: { cellMetadataId: currentCellMetadataId },
+      retry_params: { clusteringShouldRun },
```

Comment:
At the moment, in `master` branch do we use `retry_params` for anything in the qc pipeline? Or is it a new field that you are introducing?

Reply:
the field is used in gem2s and seurat